daffodil-core/src/main/scala/org/apache/daffodil/compiler/Compiler.scala

```diff
@@ -255,7 +255,7 @@ class Compiler(var validateDFDLSchemas: Boolean = true)
 
   def reload(savedParser: File) = reload(new FileInputStream(savedParser).getChannel())
 
-  def reload(savedParser: DFDL.Input): DFDL.DataProcessor = {
+  def reload(savedParser: java.nio.channels.ReadableByteChannel): DFDL.DataProcessor = {
```

Comment:
Should this be deprecated, i.e., the new real API wants InputStream now right? I think the only reasons for wanting a Channel were to allow memory mapped I/O, which was a false requirement.

Reply:
The save/reload method weren't change, only the parse method, and that was necessary to support multiple calls to parse() using the same input. There isn't really the same motivaction here since it's just saving/reloading parsers. But it might be nice for consistency? I'll create a bug think about changing this.